addons/website_sale_stock/tests/test_website_sale_stock_multilang.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleStockMultilang(HttpCase):
+    def test_website_sale_stock_multilang(self):
+        # Install French
+        website = self.env.ref('website.default_website')
+        lang_fr = self.env['res.lang']._activate_lang('fr_FR')
+        website.language_ids = self.env.ref('base.lang_en') + lang_fr
+        website.default_lang_id = self.env.ref('base.lang_en')
```

Comment:
Just this should be enough?
```suggestion
        website.language_ids = [Command.link(lang_fr.id)]
```